src/components/organisms/BannerAdmin/BannerAdmin.tsx

```diff
@@ -2,14 +2,15 @@ import React, { useState, useCallback, useRef } from "react";
 
 import { makeUpdateBanner } from "api/bannerAdmin";
 
-import { currentVenueSelector } from "utils/selectors";
+import { AnyVenue } from "types/venues";
 
-import { useSelector } from "hooks/useSelector";
+interface BannerAdminProps {
+  venueId?: string;
+  venue: AnyVenue;
+}
```

Comment:
Since we're passing in `venue`, is there any need for `venueId`? If the intent here was that `AnyVenue` doesn't have a `venueId`, then we should make it `WithId<AnyVenue>`